docs/src/components/markdown.js

```diff
@@ -127,3 +146,80 @@ export const Markdown = styled.article`
     left: 1em;
   }
 `;
+
+const scope = {
+  useGravity,
+  useFriction,
+  useFluidResistance,
+  useGravity2D,
+};
+
+export const Markdown = ({ html = null }) => {
+  const content = React.useRef(null);
+
+  const findPlayground = React.useCallback(className => {
+    if (content.current) {
+      const elements = content.current.getElementsByClassName(className);
+      return elements;
+    }
+  }, []);
+
+  const mountContainer = React.useCallback((source, preview) => {
+    return (
+      <div className="Interactive">
+        <ThemeProvider theme={theme}>
+          <LiveProvider code={source} scope={scope} theme={NightOwl}>
+            <StyledContainer splitVertical>
+              {preview && <StyledPreview splitVertical />}
+              <StyledError />
+              <StyledEditor />
+            </StyledContainer>
+          </LiveProvider>
+        </ThemeProvider>
+      </div>
+    );
+  }, []);
+
+  const renderPlaygrounds = React.useCallback(() => {
+    const playgrounds = Array.prototype.slice.call(
+      findPlayground('language-playground'),
+      0
+    );
+
+    for (const p in playgrounds) {
+      if (playgrounds.hasOwnProperty(p)) {
+        const source = playgrounds[p].textContent;
+        ReactDOM.render(
+          mountContainer(source, true),
+          playgrounds[p].parentNode
+        );
+      }
+    }
+
+    const [playgroundsNoRender] = findPlayground(
```

Comment:
I don't think the code related to noRender will be useful for you since you're using react-live. This refers to something specific in `component-playground`: https://github.com/FormidableLabs/component-playground#norender